mozilla-lockwise/lockwise-ios
# delete from edit screen

Fixes #938

To test:

1. Log in
1. Tap on a login
1. Tap Edit
1. Tap Delete
1. Tap Confirm Dialog
1. Verify that the login has been deleted
